src/applications/personalization/dashboard/containers/PrescriptionsWidget.jsx

```diff
@@ -24,7 +24,7 @@ function recordDashboardClick(product) {
 
 class PrescriptionsWidget extends React.Component {
   componentDidMount() {
-    if (!this.props.loading) {
+    if (this.props.canAccessRx) {
       this.props.loadPrescriptions({ active: true, sort: '-refill_submit_date' });
     }
```

Comment:
The user's profile is already loaded once the widgets mount, because they're rendered inside a `RequiredLoginView`.